fathom/src/surface/elaboration.rs

```diff
@@ -965,17 +965,73 @@ impl<'interner, 'arena> Context<'interner, 'arena> {
 
                 core::Term::RecordLit(range.into(), labels, exprs.into())
             }
-            (Term::UnitLiteral(range), Value::Universe) => {
-                core::Term::RecordType(range.into(), &[], &[])
-            }
-            (Term::UnitLiteral(range), _)
-                if matches!(
-                    expected_type.match_prim_spine(),
-                    Some((Prim::FormatType, [])),
-                ) =>
+            (Term::Tuple(range, elem_exprs), _)
+                if elem_exprs.is_empty()
+                    && matches!(
+                        expected_type.match_prim_spine(),
+                        Some((Prim::FormatType, [])),
+                    ) =>
             {
                 core::Term::FormatRecord(range.into(), &[], &[])
             }
+            (Term::Tuple(range, elem_exprs), Value::Universe) => {
+                let labels = (0..elem_exprs.len())
+                    .map(|idx| self.interner.borrow_mut().get_or_intern(format!("_{idx}")));
```

Comment:
Looks CI failed because our minimum supported Rust version doesn't support the format string shorthand. If you could tweak that to use the more verbose form it should fix it.